### PR TITLE
Adding 2 new books

### PR DESCRIPTION
**Pull request by sger**
1) Go Design Patterns
2) Go Reactive Programming

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Go, commonly referred to as golang, is a programming language initially develope
 
 Solve your Go problems using a problem-solution approach. Each recipe is a self-contained answer to a practical programming problem in Go. Go Recipes contains recipes that deal with the fundamentals of Go, allowing you to build simple, reliable, and efficient software. Other topics include working with data using modern NoSQL databases such as MongoDB and RethinkDB. The book provides in-depth guidance for building highly scalable backend APIs in Go for your mobile client applications and web client applications.
 
+=======
```
Comment (nathany) [inline README.md]: Not sure why this is added?

File: README.md [hunk 2]
```diff
@@ -116,7 +117,19 @@ Solve your Go problems using a problem-solution approach. Each recipe is a self-
 **Advanced Books**
 ---
 
-###[Test-driven development with Go ](https://leanpub.com/golang-tdd) *Free*
+###[Go Design Patterns](https://www.packtpub.com/application-development/go-design-patterns)
```
Comment (nathany) [inline README.md]: I think you need a space after `###` for each of these.

File: README.md [hunk 3]
```diff
@@ -116,7 +117,19 @@ Solve your Go problems using a problem-solution approach. Each recipe is a self-
 **Advanced Books**
 ---
 
-###[Test-driven development with Go ](https://leanpub.com/golang-tdd) *Free*
```
Comment (nathany) [inline README.md]: Free books go first. Please add books at the end of the section, not at the top.

File: README.md (hunk 2 above)
Comment (sger) [inline README.md]: something is bad with the formatting of markdown.I will open a new PR sorry

**Pull request closed without merging**